server/api/rating/types.d.ts

```diff
@@ -0,0 +1,7 @@
+import { type IMDBRating } from '@server/api/rating/imdbRadarrProxy';
+import { type RTRating } from '@server/api/rating/rottentomatoes';
+
+export interface RatingResponse {
+  rt?: RTRating;
+  imdb?: IMDBRating;
+}
```

Comment:
Let's follow the same pattern as the other type files. Create a ratings.ts file in the root of `server/api` instead of a declaration file like this.